src/System.Private.CoreLib/shared/System/Diagnostics/StackTrace.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Please also annotate StackTrace.CoreCLR.cs as part of this PR.  Same goes for any other cases where there are partial files: we should annotate all parts as part of the same go.